frontend/src/types/budget.ts

```diff
@@ -33,3 +49,8 @@ export interface Category {
   parentCategory?: Category;
   createDatetime: Date | string;
 }
+
+export interface CategoryOption {
```

Comment:
This is one-off type for our form selector. Let's only add reusable / meaningful types in this file